ui/src/tall_info.test.tsx

```diff
@@ -73,4 +74,31 @@ describe('TallInfo.tsx', () => {
     expect(pushMock).toHaveBeenCalled()
     expect(wave.args[name]).toBe(name)
   })
+
+  it('Makes card clickable only if name was specified but label was not', () => {
+    const { getByTestId } = render(<View {...tallInfoProps} />)
+    fireEvent.click(getByTestId(name))
+    expect(pushMock).toHaveBeenCalled()
+    expect(wave.args[name]).toBe(name)
+  })
+
+  it('Makes card unclickable if name was not provided', () => {
```

Comment:
Name is required, so it always has to be provided.
```suggestion
  it('Makes card unclickable if name is empty', () => {
```